### PR TITLE
feat: Added videoListing page and authentication

### PR DESCRIPTION
**Pull request by Hardiegogo**
- Created AuthContext 
- Added videoListing page

Test credentials:
email: chirag.gps@gmail.com
password: chirag

Screenshots:
image

login page:
image

After logging in:
image

### CODE REVIEW

File: src/reducers/authReducer.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+export const authReducer=(state,action)=>{
+    switch(action.type){
+        case "SIGNUP_USER":
+        case "LOGIN_USER":
+            console.log(action)
+            return{
```
Comment (ashishnnnnn) [inline src/reducers/authReducer.js:5]: This console can be removed..if not required..

**Review by ashishnnnnn — APPROVED** [review]
Code looks fine..
Like the use of Chakra Ui.

**Review by aman11s — APPROVED** [review]
Everything looks fine. Love the UI

File: src/reducers/authReducer.js (hunk 1 above)
Comment (Hardiegogo) [reply src/reducers/authReducer.js:5]: Got it.

**Pull request merged**